module-domain/src/main/kotlin/site/yourevents/guest/port/in/OwnerNicknameUseCase.kt

```diff
@@ -0,0 +1,12 @@
+package site.yourevents.guest.port.`in`
+
+import site.yourevents.guest.domain.Guest
+import java.util.UUID
+
+interface OwnerNicknameUseCase {
+    fun createOwnerNickname(
+        memberId: UUID,
+        invitationId: UUID,
+        nickname: String
+    ): Guest
+}
```

Comment:
각 파일 마지막에 개행이 없으면, GitHub에서 EOF 관련하여 🚫 경고가 뜹니다..ㅎㅎ
해당 경고가 있는 파일에 대해 전부 개행 추가 부탁드립니다!
```suggestion
}

```

Reply:
아 알겠습니다!! 전부 개행 처리 진행하였습니다!!